app/graph/nodes/answer/struct_db_node.py

```
@@ -0,0 +1,181 @@
+                persona_lines.append(f"Your name is {name}.")
+            if style:
+                persona_lines.append(
+                    f"[Speech style instruction — written in Korean]: {style}\n"
+                    f"→ Translate the above Korean style instruction into {lang_name} first, "
+                    f"then follow it exactly in {lang_name}. "
+                    f"If it says to add a word/phrase at the end of sentences, "
+                    f"translate that word/phrase into {lang_name} and add it.\n"
+                    f"- CRITICAL: The style MUST be visible in your response.\n"
+                    f"- CRITICAL: Do NOT use the original Korean words — always translate them into {lang_name}."
+                )
+            persona_block = "\n".join(persona_lines) if persona_lines else f"You are a friendly tourist guide assistant."
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)